lib/jekyll/filters.rb

```diff
@@ -363,6 +372,7 @@ def item_property(item, property)
       end
     end
 
+    private
```

Comment:
These are all just helper methods that [were already private](https://github.com/jekyll/jekyll/blob/v3.1.6/lib/jekyll/filters.rb#L314) :+1: